src/index.js

```diff
@@ -26,8 +27,18 @@ module.exports = {
     serialize: (data, cb) => {
       cb(null, data)
     },
-    cid: (data, cb) => {
-      cb(null, new CID(1, 'raw', data))
+    cid: (data, options, cb) => {
+      if (options instanceof Function) {
+        cb = options
+        options = {}
+      }
+      options = options || {}
+      const hashAlg = options.hashAlg || 'sha2-256'
+      const version = options.version || 1
```

Comment:
This is not correct. If `options.version` is set to `0`, it will result in `version == 1` as `0` evaluates to `false`.